src/core.c/REPL.rakumod

```diff
@@ -315,6 +331,10 @@ do {
         }
 
         method init(Mu \compiler, $multi-line-enabled --> Nil) {
+            if not $*IN.t {
```

Comment:
s/if not/unless/  ?

Reply:
I've been in Java/Kotlin land too long!

Reply:
Fixed. Though I prefer leaving it as a statement+block rather than as a statement modifier in this instance.